docs/user/ppl/functions/datetime.rst

```diff
@@ -817,6 +897,31 @@ Example::
     +--------------------------+
 
 
+HOUR_OF_DAY
+----
```

Comment:
```suggestion
HOUR_OF_DAY
-----------
```

Reply:
Fix this everywhere please

Reply:
fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/281/commits/7b927df652fd09ef0e74315eb6a2abfbffe1f8f3